velox/exec/tests/ExchangeClientTest.cpp

```diff
@@ -122,17 +123,23 @@ class ExchangeClientTest : public testing::Test,
     return std::make_unique<SerializedPage>(std::move(ioBuf));
   }
 
+  folly::Executor* executor() const {
+    return GetParam() ? executor_.get() : nullptr;
```

Comment:
Can we allow only one mode: with the executor?

Reply:
Yes, changed. Thanks!